Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs

```diff
@@ -143,6 +143,21 @@ public Task UpdateStatusByIdsAsync(IEnumerable<Guid> ids, Guid projectId, Status
             return GetCollection().UpdateManyAsync(filter, updateDefinition, updateOptions);
         }
 
+        public Task BulkUpdateStatusTestAsync(IEnumerable<Guid> candidatesIds, Guid projectId, Status status)
```

Comment:
Remove word Test, it's universal method for different Statuses

Reply:
done